whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapConformance.java

```diff
@@ -0,0 +1,15 @@
+package net.ripe.db.whois.api.rdap;
+
+public enum RdapConformance {
+    CIDR_0("cidr0"), LEVEL_0("rdap_level_0"), FLAT_MODEL("nro_rdap_profile_asn_flat_0");
```

Comment:
Do we **also** need nro_rdap_profile_0 ? Or do we not meet the requirements yet? Is that mentioned in the document?

I just found the extension registry: https://www.iana.org/assignments/rdap-extensions/rdap-extensions.xhtml

Reply:
To signify conformance with this specification, the string literal
   "nro_rdap_profile_0" MUST be included as an rdapConformance value in
   each response.